### PR TITLE
Feature/conference stubs 16

### PR DESCRIPTION
**Pull request by chenry**
This pull request fulfills #16 

Specifically, there is now an endpoint **/conference** that will retrieve a mock list of conference view objects with their associated location.

This can be tested by bringing up the application and hitting the **/conference** url.  This only does a GET.  POST, UPDATE and DELETE operations are not yet supported.

### CODE REVIEW

**Comment by mccrackend** [comment]
Builds, runs, spits out lots of stub.  LGTM.

One thing we should try to address in a future PR is the outbound DateTime format.  I'll create an issue to discuss that proposal.
eg. 2012-04-23T18:25:43
[Source](http://stackoverflow.com/questions/10286204/the-right-json-date-format)

File: src/main/java/hello/Application.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
 import org.springframework.context.annotation.ComponentScan;
 
-@ComponentScan
+@ComponentScan(basePackages = {"hello", "com.ioextendedgr.web"})
```
Comment (mccrackend) [inline src/main/java/hello/Application.java:7]: Can we move this Application.java class out of this /hello/ package?

Comment (mccrackend) [inline src/main/java/hello/Application.java:7]: Nevermind. I already created an issue to do all this cleanup..

**Comment by chenry** [comment]
@mccrackend When you merged this in and brought the branch back, was there something missing that you were looking for as to not close out the #16 ?

**Comment by mccrackend** [comment]
Weird. Not sure why it didn't auto close the issue - i thought it normally did that.

**Pull request merged**